site/content/entry/ldn-is-here/index.md

```diff
@@ -0,0 +1,159 @@
+In order to get together with other players, you’re going to have to join the same room with yuzu’s room browser. 
+Out of the box, we provide you with a bunch of public rooms hosted across the world by yuzu.
+Creating and joining rooms is extremely easy in yuzu and can be done in just a few clicks.
+
+{{< imgs "./room_browser.png" >}}
+
+If you’re a verified user, you can create a public room for people to join. 
+These public rooms can be seen by anyone using the room browser, but you can also put a password on them to restrict users. 
+Remember to port forward, otherwise your friends won’t be able to connect!
+
+Unverified users aren’t left without options, though - they still have the ability to create unlisted rooms, direct connecting, and can join any yuzu hosted room.
+
+Do note that verified users will have their privileges revoked for violating any site policies while on the room chatroom. 
+Please respect the "recommended game" listed in publicly hosted rooms, as even unrelated games will add to the bandwidth load.
+
+{{< imgs "./room_chat.png" >}}
+
+**Note: You can use these rooms for LAN mode games too - instead of ZeroTier or Hamachi.**
+
+## Development
+
+FearlessTobi began the development by leveraging the existing virtual rooms infrastructure from [Citra](https://citra-emu.org).
+Due to code similarities between yuzu and Citra, this tried and tested infrastructure easily became the foundation of this feature.
+
+Now that the virtual rooms were taken care of, Tobi moved onto the `LDN` service and its protocols.
+Using the research and code from the [ldn_mitm](https://github.com/spacemeowx2/ldn_mitm) project, he implemented the necessary changes for yuzu instances to communicate with each other over these virtual rooms.
+
+As the feature slowly matured, Tobi began leveraging yuzu's internal team of testers to test the feature across several titles for compatibilty and performance validation. 
+And to prevent user IPs from being leaked when connected to rooms, Tobi changed the implementation to use fake IPs for routing.
+
+With the multiple testing iterations, various bugs and glitches were found. 
+[german77](https://github.com/german77) relied on his own reverse-engineering and hardware tests, to debug these issues and fixed them all.
+He also made several UI fixes to improve its usability and brought the codebase up to yuzu standards.
+
+## Compatibility
+
+Please note that as this is the initial release of the feature, some games might have some limitations.
+We hope to slowly fix these over the course of next few months.
+
+yuzu's Local Wireless Multiplayer was tested by our testers across a small set of games and their corresponding compatibilty is as follows:
+
+{{< imgs "./compat.png" >}}
+
+
+## Screenshots
+
+{{< single-title-imgs
+    "Animal Crossing: New Horizons"
+    "./acnh1.jpg"
+    "./acnh2.png"
+>}}
+
+{{< single-title-imgs
+    "Pokémon Legends: Arceus"
+    "./arceustrade1.png"
+    "./arceustrade2.png"
+>}}
```

Comment:
```suggestion
{{< single-title-imgs
    "Pokémon Legends: Arceus"
    "./arceustrade1.png"
    "./arceustrade2.png"
>}}
```
To match the rest of the screenshots of this section, please crop the screenshots to just the gameplay screen and re-upload.